### DISCUSSION TITLE
Introducing the Matterbridge zigbee2MQTT plugin

### DISCUSSION THREAD

**Discussion by Luligu** [Koenkk/zigbee2mqtt#22147] [Show and tell]
Hello Zigbee2MQTT Community,

I'm excited to share a project I've been working on, https://github.com/Luligu/matterbridge-zigbee2mqtt, which I believe many of you will find useful in your smart home setups.

The Matterbridge zigbee2mqtt plugin is the official Matterbridge zigbee2mqtt plugin. It exposes all zigbee2mqtt devices and groups to Matter.

No hub or dedicated hardware is needed.

Matterbridge is the matter.js plugin manager (https://github.com/Luligu/matterbridge). 

Matterbridge has been extensively tested with Apple Home and Home Assistant but, since it is fully Matter compliant, it should work on all Matter-enabled ecosystems.

Getting Started:
The project is published on npm and available on GitHub. You can find detailed instructions on how to set it up and use it in your own environment. I've put effort into making the documentation clear and helpful, but if you have questions or run into issues, feel free to raise them in the repository's Issues section.

Feedback and Contributions:
I'm looking for feedback to improve and expand the project. Whether it's a feature request, a bug report, or a suggestion for improvement, your input is valuable. Contributions to the code are also welcome if you're inclined to do so!

Join the Journey:
I believe this project has the potential to grow and become an integral part of our Zigbee2MQTT experiences. Your support, feedback, and contributions can shape its future.

Thank you for checking out https://github.com/Luligu/matterbridge-zigbee2mqtt, and I can't wait to see how it helps in your smart home adventures!

Best regards,
Luligu

**Comment by Koenkk** [comment]
Really nice project! Do you see anything in integrating this directly into z2m as an [extension](https://github.com/Koenkk/zigbee2mqtt/tree/master/lib/extension)?

**Comment by Luligu** [comment]
Hi, thank you so much. I really appreciate your comment.
I know how to code the extensions: I already published month ago:
https://github.com/Luligu/zigbee2mqtt-automations.
Matterbridge is light weight but inside z2m like an extension I'm not sure... but is a really interesting idea.
Eventually I could modify the plugin and create an autonomous instance without Matterbridge that can load faster inside z2m like an extension.  
I will understand if it is possible and get back to you. 
I was wondering if you can link Matterbridge zigbee2mqtt on https://www.zigbee2mqtt.io/ like the official zigbee2mqtt plugin of Matterbridge :D

**Reply by Koenkk** [reply]
No problem, feel free to make a PR to add it to the docs!

**Comment by Luligu** [comment]
For the extension I will need to ask you a few questions.
Can I join you on Discord?
Thank you.

**Reply by Koenkk** [reply]
I've send you a friend request
